.github/workflows/release-drafter.yml

```diff
@@ -6,8 +6,13 @@ on:
     branches:
       - master
 
+permissions: {}
 jobs:
   update_release_draft:
+    permissions:
+      pull-requests: write  #  to add label to PR (release-drafter/release-drafter)
```

Comment:
Does the drafter need this permission? We don't use it to add permissions, rather rely on the permissions set in the PR. WDYT?